Homework1/ComplexStruct.UnitTests/UnitTest1.cs

```diff
@@ -0,0 +1,44 @@
+using NUnit.Framework.Constraints;
+using System.Numerics;
+
+namespace ComplexStruct.UnitTests
+{
+    [TestFixture]
+    public class ComplexTests
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var complex = new Complex(1, 2);
+
+            Assert.That(complex.Re, Is.EqualTo(1));
+            Assert.That(complex.Im, Is.EqualTo(2));
+
+        }
+        [TestCase(1, 2, "1 + 2i")]
```

Comment:
Тестов метода `ToString()` надо было сделать несколько, протестировав разные способы записи, например
```suggestion
        [TestCase(1, 2, "1 + 2i")]
        [TestCase(1, 1, "1 + i")]
        [TestCase(1, -1, "1 - i")]
        [TestCase(1, 0, "1")]
        [TestCase(0, 2, "2i")]
```